src/components/access-type.js

```diff
@@ -9,28 +9,31 @@ import {
   METER_UPDATED,
   ASSET_PLANS,
   CAMPAIGN_SUBSCRIPTION_GROUP_UPDATED,
-} from '../store/actions';
-import PropTypes from 'prop-types';
-import {awaitHelper} from '../utils';
-
-const prod_Host = 'https://www.accesstype.com';
-const staging_Host = 'https://staging.accesstype.com';
+} from "../store/actions";
+import PropTypes from "prop-types";
+import { awaitHelper } from "../utils";
 
 class AccessTypeBase extends React.Component {
+  constructor(props) {
+    super(props);
+    this.prod_Host = props.prodHost || "https://www.accesstype.com";
```

Comment:
can you change the casing ?

Reply:
This is how it has been used the all over the places before so i kept the same.
But sure we can make it camel case